vaccine_feed_ingest/ingestors/arcgis.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
```

Comment:
Since there are multiple arcgis endpoints, I might lean toward calling this arcgis_featureserver or something, but do not feel strongly.